### PR TITLE
move arm to post_init_configuration after successful initialization

### PR DESCRIPTION
**Pull request by benmaidel**
ref: https://github.com/mojin-robotics/kevin/issues/221

depends on:
- https://github.com/mojin-robotics/mojin_robots/pull/432

TODO:
- [x] test on hardware

### CODE REVIEW

File: precise_driver/src/precise_hw_interface.cpp [hunk 1]
```diff
@@ -123,8 +133,20 @@ namespace precise_driver
...
+                        ret &= device_->waitForEom();
+                        if(!ret)
+                        {
+                            std::string msg = "Failed to move to post init configuration";
+                            ROS_ERROR_STREAM_NAMED("precise_hw_interface",msg);
+                        }
+                    }
+
                     device_->startCommandThread();
                     std::string msg = "successfully initialized";
```
Comment (fmessmer) [inline precise_driver/src/precise_hw_interface.cpp:151]: you could add some info to this string so we know in the res.message whether the post_init movement worked, e.g.
- `successfully initialized` when no post_init_config is set
- `successfully initialized and moved to post_init_configuration` when init and movement successful
- `successfully initialized but failed to move to post_init_configuration` when init worked, but movement failed

**Review by fmessmer — APPROVED** [review]

File: precise_driver/config/precise_controllers.yaml [hunk 2]
```diff
@@ -26,6 +26,8 @@ hardware_interface:
   in_range: -1
   straight: 0 #0=rotational; -1=straight
 
+  post_init_configuration: [0.085, -1.552, 3.136, 3.155, 0.038] #config where the arm should move to after successful initialization
```
Comment (fmessmer) [inline precise_driver/config/precise_controllers.yaml:29]: how about a mapping of joints and values...then you could also only move selected joints and leave the others where they are...

Comment (benmaidel) [reply precise_driver/config/precise_controllers.yaml:29]: ok, could be an additional feature. For know I think it's ok to give a complete arm configuration. Then we know that the arm has a valid configuration after init. If we find a need for that feature in future we can add it...

**Comment by benmaidel** [comment]
tested with mrk-5 :heavy_check_mark:

**Pull request merged**